x-pack/plugins/security_solution/public/overview/links.ts

```diff
@@ -109,3 +111,24 @@ export const ecsDataQualityDashboardLinks: LinkItem = {
     }),
   ],
 };
+
+export const coverageOverviewDashboardLinks: LinkItem = {
+  id: SecurityPageName.coverageOverview,
+  title: COVERAGE_OVERVIEW,
+  landingImage: overviewPageImg, // TODO: change with updated image before removing feature flag https://github.com/elastic/security-team/issues/2905
+  description: i18n.translate(
+    'xpack.securitySolution.appLinks.coverageOverviewDashboardDescription',
+    {
+      defaultMessage:
+        'An overview of rule coverage according to the MITRE ATT&CK\u00AE specifications',
+    }
+  ),
+  path: COVERAGE_OVERVIEW_PATH,
+  capabilities: [`${SERVER_APP_ID}.show`],
+  globalSearchKeywords: [
+    i18n.translate('xpack.securitySolution.appLinks.coverageOverviewDashboard', {
+      defaultMessage: 'MITRE ATT&CK Coverage',
+    }),
+  ],
+  experimentalKey: 'detectionsCoverageOverview',
```

Comment:
@banderror This is the place the page we determine render based on the feature flag

Reply:
Thanks for pointing to this @dplumlee. I went ahead and started testing the PR. I don't have the `detectionsCoverageOverview` flag set in my local kibana config, and I don't see the link to this page in the `Security -> Dashboards` menu:

<img width="585" alt="Screenshot 2023-06-27 at 19 32 22" src="https://github.com/elastic/kibana/assets/7359339/52bb9050-6744-4a7d-a9de-14d0a8732b56">

However, I'm still able to open the page by hitting its URL directly: `http://localhost:5601/kbn/app/security/rules_coverage_overview`

<img width="1066" alt="Screenshot 2023-06-27 at 19 33 10" src="https://github.com/elastic/kibana/assets/7359339/87804fcf-a45c-4632-ad62-c314b334994d">

I think we should conditionally register this page in the FE router only if the flag is set.

We should start getting used to being extremely careful with feature toggling and what we hide behind a feature flag, to avoid releasing unfinished features or breaking existing code in production in the (very close) future when the Serverless CD pipeline becomes our primary release process. We will be talking about it more in the next release cycle.

cc @maximpn @xcrzx